react/components/ProductSummaryImage/ProductImage.js

```diff
@@ -155,19 +155,19 @@ const ProductImageContent = ({
     legacyContainerClasses
   )
 
-  if (!sku || hasError) {
+  const images = pathOr([], ['images'], sku)
+  const hoverImage = findImageByLabel(images, hoverImageLabel)
+
+  let imageUrl = pathOr('', ['image', 'imageUrl'], sku)
```

Comment:
```suggestion
  let imageUrl = sku?.image?.imageUrl ?? ''
```

Reply:
We don't support this features to `js`, so I think we should only try to make this change when we pass these files to `tsx`

Reply:
That's why I'm not removing ramda too